assets/styles/main.css

```
@@ -3,15 +3,26 @@
     "Segoe UI", Arial, Helvetica, "Helvetica Neue", sans-serif;
 }
 
+body {
+  padding-bottom: 3rem;
+}
+
 ul {
   list-style: none;
-  padding-left: 20px;
+  padding-left: 10px;
 }
 
 ul > li {
   margin: 0.5rem 0;
   font-size: 1.3rem;
 }
+.indent {
+  text-indent: 1rem; /* Adjust the value as needed */
+}
```

This is incorrect. You don't need `text-indent`, you need `margin-left`.

The list items don't have a list icon because of the `list-style: none;` CSS rule applied to all the `ul` elements. To have one enabled for the indented items, add the `list-style: initial` rule.

```suggestion
.indent {
  margin-left: 1rem;
  list-style: initial;
}
```